opentext-page.html

```diff
@@ -0,0 +1,499 @@
+<DOCTYPE html>
+<head>
+    <title>OpenText</title>
+        <style>
+            ul {
+                list-style-type: none;
+            }
+            li {
+                display: inline;
+            }
+            .nav {
+                float: right;
+            }
+            .container {
+                width: 16.6%;
+            }
+            .drop-down-menu  li {
+                display: block;
+            }
+            /* .slide-class */
+            /* dots to indicate which image you are on */
+            .in-slide {
+                width: 50%;
+                float: right;
+                margin-top: -70px;
+            }
+            .container-social {
+                width: 25%;
+                float: left;
+            }
+            .container-2 {
+                width: 33.3%;
+                float: left;
+            }
+            .container-3 {
+                margin-top: 70px;
+                width: 25%;
+                display: inline;
+                float: left;              
+            }
+            .column {
+            float: left;
+            width: 20%;
+        }s
```

Comment:
Indentation issue with the } ... this needs to align; and it also throws off the alignment of the css selectors on lines 45-71